overlord/fdestate/fdemgr.go

```diff
@@ -49,12 +54,31 @@ func Manager(st *state.State, runner *state.TaskRunner) *FDEManager {
 	return m
 }
 
+// Ensure implements StateManager.Ensure
 func (m *FDEManager) Ensure() error {
 	return nil
 }
 
-func (m *FDEManager) resealKeyForBootChains(method device.SealingMethod, rootdir string, params *boot.ResealKeyForBootChainsParams, expectReseal bool) error {
-	return backend.ResealKeyForBootChains(method, rootdir, params, expectReseal)
+// StartUp implements StateStarterUp.Startup
+func (m *FDEManager) StartUp() error {
+	m.state.Lock()
+	defer m.state.Unlock()
+	return initializeState(m.state)
+}
+
+func (m *FDEManager) resealKeyForBootChains(unlocker boot.Unlocker, method device.SealingMethod, rootdir string, params *boot.ResealKeyForBootChainsParams, expectReseal bool) error {
```

Comment:
That is some optimization. Because sealing has to has a lot of binaries, several times, on some machines it is very slow. And that blocks the whole state and some requests on the server API get timed out. We will get rid of it once we have look at  task conflicts/locking fde state. But at this point we have to keep it in.